src/topology/algebra/ordered.lean

```diff
@@ -1437,46 +1438,6 @@ end
 
 end linear_ordered_field
 
-section discrete_linear_ordered_field
-variables [discrete_linear_ordered_field α] [topological_space α] [order_topology α]
-
-/-- The function `x ↦ x⁻¹` tends to `+∞` on the right of `0`. -/
-lemma tendsto_inv_zero_at_top : tendsto (λx:α, x⁻¹) (𝓝[set.Ioi (0:α)] 0) at_top :=
```

Comment:
Why did you delete this section?

Reply:
I'm sorry, that must have been an accident when I was editing in order to merge. I'll restore now.